lib/teiserver/matchmaking/queue_server.ex

```diff
@@ -23,15 +23,27 @@ defmodule Teiserver.Matchmaking.QueueServer do
   @type id :: String.t()
 
   @typedoc """
-  Queue statistics for monitoring
+  Internal queue statistics (without player_count)
   """
-  @type stats :: %{
+  @type internal_stats :: %{
           total_joined: non_neg_integer(),
           total_left: non_neg_integer(),
           total_wait_time_s: non_neg_integer(),
           total_matched: non_neg_integer()
         }
 
+  # Queue statistics for monitoring (includes player_count)
+  @typep stats :: %{
+           total_joined: non_neg_integer(),
+           total_left: non_neg_integer(),
+           total_wait_time_s: non_neg_integer(),
+           total_matched: non_neg_integer(),
+           player_count: non_neg_integer()
+         }
+
+  # Public type alias for external modules to reference
+  @type queue_stats :: stats()
```

Comment:
There is no need for this alias since it's the same type, might as well declare `stats` as public already.